kanidm_proto/src/utils.rs

```diff
@@ -0,0 +1,14 @@
+/// Shows the version string and current git commit status at build
+pub fn show_version(name: &str) {
+    println!("{}", get_version(name));
+}
+
+pub fn get_version(name: &str) -> String {
+    let version = env!("CARGO_PKG_VERSION");
+    let lgc = match last_git_commit::LastGitCommit::new().build() {
+        Ok(value) => value.id().short(),
+        Err(_) => String::from("Unknown git commit"),
```

Comment:
This will always display "unknown git commit" when packaged, so maybe something more subtle like a `--` ?

Reply:
I was going for something that was baked in, like the CARGO_PKG_VERSION, which... will need more testing for sure.

Reply:
I wonder if I can sneak it in through a call in `build.rs`